app/middleware/middleware_healthcheck.rb

```diff
@@ -0,0 +1,36 @@
+class MiddlewareHealthcheck
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    if env['PATH_INFO'.freeze] == '/status'.freeze
+      response = [503, { 'Content-Type' => 'application/json' }]
+
+      # checks the database health
+      begin
+        db_healthy = ActiveRecord::Migrator.current_version != 0
+      rescue StandardError
+        db_healthy = false
+      end
+
+      # checks the health of launchpad
+      cmr_client = Cmr::Client.client_for_environment(Rails.configuration.cmr_env, Rails.configuration.services)
+      launchpad_healthy = true
+      begin
+        timeout(2) { launchpad_healthy = cmr_client.launchpad_healthcheck.body == 'OK'.freeze }
```

Comment:
This timeout seems really short. I would suggest increasing it to at least 10 or 15s. What do you think is appropriate, @jceaser?